src/bots/filler.ts

```diff
@@ -95,6 +98,7 @@ import { LRUCache } from 'lru-cache';
 import { bs58 } from '@project-serum/anchor/dist/cjs/utils/bytes';
 import { PythPriceFeedSubscriber } from '../pythPriceFeedSubscriber';
 import { TxThreaded } from './common/txThreaded';
+import { NodeToTriggerWithMakers } from 'src/experimental-bots/filler-common/types';
```

Comment:
i think you need to use relative path for imports, ive seen errors in prod for these abs paths before

Reply:
👍🏽